src/components/contacts/ContactAvatar.js

```diff
@@ -93,12 +94,16 @@ const sizeConfigs = colors => ({
         ],
     textSize: 'large',
   },
+  smallest: {
+    dimensions: 16,
+    textSize: 'micro',
+  },
 });
 
 const ContactAvatar = ({ color, size = 'medium', value, ...props }) => {
+  const { isDarkMode } = useColorMode();
```

Comment:
may want to make sure this works everywhere as expected

Reply:
it should - only change should be that if a ColorModeProvider is used to set a specific colorMode in some part of the app, this version of isDarkMode will respect that (e.g. in the NFT expanded state)